mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-authority-records.feature

```diff
@@ -0,0 +1,166 @@
+    * callonce login testAdmin
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json'  }
+    * def utilFeature = 'classpath:spitfire/mod-quick-marc/features/setup/import-record.feature'
+
+    * def testInstanceId = karate.properties['instanceId']
+    * def testAuthorityId = karate.properties['authorityId']
+
+  # ================= positive test cases =================
+
+  Scenario: Edit quick-marc record tags
+    Given path 'records-editor/records'
+    And param externalId = testAuthorityId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And def record = response
+    And def tag = karate.jsonPath(record, "$.fields[?(@.tag=='551')]")[0]
+
+    * def newTagContent = '$a Updated Content'
+    * set tag.content = newTagContent
+    * remove record.fields[?(@.tag=='551')]
+    * record.fields.push(tag)
+    * set record.relatedRecordVersion = 2
+
+    Given path 'records-editor/records', record.parsedRecordId
+    And headers headersUser
+    And request record
+    When method PUT
+    Then status 202
+
+    Given path 'records-editor/records'
+    And param externalId = testAuthorityId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And match karate.jsonPath(response, "$.fields[?(@.tag=='551')]")[0].content == newTagContent
+
+  Scenario: Edit quick-marc record delete not required tag
+    Given path 'records-editor/records'
+    And param externalId = testAuthorityId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And def record = response
+
+    * remove record.fields[?(@.tag=='551')]
+    * set record.relatedRecordVersion = 3
+
+    Given path 'records-editor/records', record.parsedRecordId
+    And headers headersUser
+    And request record
+    When method PUT
+    Then status 202
+
+    Given path 'records-editor/records'
+    And param externalId = testAuthorityId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And match response.fields[?(@.tag=='551')] == []
+
+    Given path '/source-storage/source-records'
+    And param recordType = 'MARC_AUTHORITY'
+    And headers headersUser
+    When method get
+    Then status 200
+    And match response.sourceRecords[0].parsedRecord.content.fields[*].551 == []
+
+    Given path 'authority-storage/authorities', testAuthorityId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And match response.saftGeographicName == []
+
+  Scenario: Edit quick-marc record add new tag, should be updated in SRS
+    Given path 'records-editor/records'
+    And param externalId = testAuthorityId
+    And headers headersUser
+    When method GET
+    Then status 200
+    And def record = response
+
+    * def fields = record.fields
+    * def newField = { "tag": "550", "content": "$z Test tag", "indicators": [ "\\", "\\" ], "isProtected":false }
```

Comment:
`isProtected` field is not required, miss it.

Reply:
`isProtected` field is needed, because when comparing it is in response, we also use it in holdings karate tests.